src/azure-cli-core/azure/cli/core/intercept_survey.py

```diff
@@ -0,0 +1,83 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import sys
+import json
+from datetime import datetime, timedelta
+from azure.cli.core._config import GLOBAL_CONFIG_DIR
+from azure.cli.core.style import print_styled_text
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+SURVEY_NOTE_NAME = 'az_survey.json'
+GLOBAL_SURVEY_NOTE_PATH = os.path.join(GLOBAL_CONFIG_DIR, SURVEY_NOTE_NAME)
+
+EXPERIENCE_PERIOD_IN_DAYS = 1
```

Comment:
```suggestion
EXPERIENCE_PERIOD_IN_DAYS = 3
```
As PM finally decided